src/normalize/entities.ts

```diff
@@ -470,12 +470,18 @@ export const Document = new Entity({
 	ct: 'contentType'
 });
 
+export const MessagePartInputForDocuments = new Entity({
```

Comment:
can we use MimePartInput ? it looks same

Reply:
or ExistingAttachmentInput

Reply:
actually we cant use , as MimePartInput or ExistingAttachmentInput uses messageId as mid for mapping , whereas in SaveDocument it uses id for messageId